packages/devtools_test/lib/src/mocks/generated_mocks_factories.dart

```diff
@@ -202,6 +204,9 @@ MockLoggingController createMockLoggingControllerWithDefaults({
   return mockLoggingController;
 }
 
+// TODO(kenz): should we be using a Testing only class with overrides here
```

Comment:
I think it might be worth creating a Fake that can re-use most of the book keeping logic. This is already a pretty involved mock at this point.

Reply:
Found a way to remove this completely.